public/index.html

```diff
@@ -1,214 +1,237 @@
-
-          <hr/>
+    <div id="map" data-enable-search="true"></div>
+  </div>
 
-        </div>
-        <div class="donation-map-title">
-          <h2 class="i18n" data-i18n= "ftm-map-of-sites">Donation Sites</h2>
-          <span id="list-stats"></span>
-        </div>
+  <div class="locations-container" style="display: none;">
+    <div class="filters-container" style="display: none;">
+      <form name="filters" class="filters-list"></form>
+    </div>
 
-        <div class="locations-loading">
-          <img src="images/icons/loader.svg" alt="" class="load-spinner"> <span class="i18n" data-i18n="ftm-donations-site-loading">Loading donation sites...</span>
-        </div>
 
-      <div class="map-container" style="display: none;">
-        <div class="row" style="margin: 30px 0px 20px 0px;">
-          <div class="offset-xs-1 col-xs-10 offset-lg-2 col-lg-8">
-            <div class="map-search-wrap">
-              <div class="search-input">
-                <input type="search" name="map-search" id="map-search" class="i18n" placeholder="Search by city, zip, or address" data-i18n="[placeholder]ftm-maps-search-placeholder">
-                <!-- Icon credit to https://feathericons.com/  -->
-                <img class="search-icon" src="images/icons/search.svg">
+    <div id="locations-list" class="locations-list"></div>
+
+    <div class="footer">
+      <div style="height:50px"></div>
+      <div class="question i18n" data-i18n="ftm-add-donation-site-question">Do you work in a hospital or clinic and need PPE? Want to update or remove information?</div>
+      <p class="i18n" data-i18n="[html]ftm-add-donation-site-answer">Please <a href="https://docs.google.com/forms/d/e/1FAIpQLSfgCpK5coPVFC6rJrE7ZhimiZuDoEaL6fo6gYqxsN_FIpJZhg/viewform">fill out this form</a>. We will review it as soon as possible.</p>
+      <div class="question i18n" data-i18n="ftm-comments-questions">Got a comment or suggestion not related to data updates? Want to help?</div>
+      <p class="i18n" data-i18n="[html]ftm-contact-info">
+        Email us at contact@findthemasks.com or join us on <a target="_blank" rel="noreferrer noopener" href="https://github.com/r-pop/findthemasks">our Github</a>.
+      </p>
+      <p class="i18n" data-i18n="[html]ftm-made-with-love">made with &lt;3 in Seattle</p>
+      <a href="javascript:;" onclick="window.location='whoweare.html'+window.location.search;" class="end i18n" data-i18n="ftm-who-we-are">who we are</a>
+      <div>
+              <div class="social-media-icons">
+                <a class="fb-xfbml-parse-ignore" target="popup" onclick="window.open('https://www.facebook.com/sharer/sharer.php?u=http%3A%2F%2Fwww.findthemasks.com%2F&amp;src=sdkpreparse','popup', 'width=600,height=800');return false;" href="https://www.facebook.com/sharer/sharer.php?u=http%3A%2F%2Fdengdeney.blogspot.com.tr%2F&amp;src=sdkpreparse">
```

Comment:
What is the blogspot URL in the href for?

Reply:
Can remove the target=popup attribute too, since you're handling it via onclick.

Reply:
Should have the same URL in href as you're opening via onclick, to handle middle- and right-button click actions consistently.

Reply:
I'll fix these -- blogspot shouldn't be there